docs/sources/trackers/google-tag-manager/tag-types/index.md

```diff
@@ -0,0 +1,302 @@
+
+</details>
+
+
+### Enhanced Ecommerce
+
+The Enhanced Ecommerce tag is used to track ecommerce events, such as product views, add to cart, and purchases.
+
+See the [Enhanced Ecommerce Tag](/docs/sources/trackers/google-tag-manager/ecommerce-tag-template/index.md) template for more information.
+
+
+### Error Tracking
+
+The Errors tracker tag provides a way to manually track any errors that occur on your website.
+
+<details plain>
+<summary>Error Tracking Parameters</summary>
+
+| **Name**   | **Required?** | **Description**                     | **Example**              |
+| ---------- | ------------- | ----------------------------------- | ------------------------ |
+| `message`  | Yes           | Error message                       | 'Cannot get user object' |
+| `filename` | No            | Filename or URL                     | 'shop.js'                |
+| `lineno`   | No            | Line number of problem code chunk   | 23                       |
+| `colno`    | No            | Column number of problem code chunk | 45                       |
+| `error`    | No            | JS `ErrorEvent`                     | {Stack trace string}     |
+
+</details>
+
+### Page View
+
+You can provide a Custom Page Title if you wish, and you can add a [custom context](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/index.md#custom-context) to the request, as also described [here](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/index.md#trackpageview).
+
+If you enable Page Activity Tracking, page pings will be sent at intervals that you can specify.
+
+The Callback Function is something you can set to a JavaScript function. If you set the callback, then instead of sending the page ping to Snowplow, the function gets invoked instead.
+
+
+### Link Click Tracking
+
+The Track Link Click event is similar to regular parameter-based events, as it lets you add parameters and track a link click as a manually encoded hit.
+
+The Enable Automatic Link Click Tracking adds listeners to the page, which will track clicks on links permitting they adhere to the blacklisted/whitelisted class names you can optionally provide.
+
+The Fix Middle-click Tracking adds a fix for some browsers where middle-clicks were not tracked properly.
+
+If you check Track HTML Content Of Clicked Link, then the full text content of the link element will be sent to Snowplow as well.
+
+Note that this tag enables the tracker's native [Link Click Tracking plugin](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/link-click/index.md), it does not directly fire an event.
+It is intended to trigger [early on a page](https://support.google.com/tagmanager/answer/7679319) to begin listening for click events, _not_ to trigger on click events like [Click Triggers](https://support.google.com/tagmanager/answer/7679320).
```

Comment:
This only applies when using the `Enable Automatic Link Click Tracking` option. You can probably move it next to that and s/this tag/this setting/.